### PR TITLE
Retrospectively documenting an unforeseen effect of HTTP router change.

### PR DESCRIPTION
**Pull request by sumerman**
## Description

A user reported change in behaviour of `/healthz` endpoint which affected their load balancing.

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [x] CHANGELOG entry for user-facing changes
- [ ] Updated the relevant documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ We use the following categories for changes:
 - The Promscale extension is now required, while the Timescaledb extension remains optional. The minimum Timescaledb version supported is now 2.6.1 [#1132], [#1297]
 - Disable running Promscale in HA and read-only simultaneously [#1254]
 - Metric tables and views are now owned by prom_admin [#1283]
+- HTTP OPTION verb is no longer supported on `/healthz` endpoint because HTTP router was changed in [#1224].
```
Comment (JamesGuthrie) [inline CHANGELOG.md:36]: I would even phrase this the other way around: The new behaviour is that _only_ GET is allowed. Previously HEAD/OPTIONS/PUT/POST would all work.

**Review by JamesGuthrie — CHANGES_REQUESTED** [review]

**Review by JamesGuthrie — APPROVED** [review]

**Comment by niksajakovljevic** [comment]
I believe this can be easily fixed. This was obviously not a planned change but a missed bug so I am not sure it should go retroactively into CHANGELOG. So maybe we can just create a bug issue instead?

**Comment by sumerman** [comment]
> I believe this can be easily fixed. This was obviously not a planned change but a missed bug so I am not sure it should go retroactively into CHANGELOG. So maybe we can just create a bug issue instead?

Whether or not we consider this a bug, it needs to be in the changelog and release notes because it already affects and confuses existing users. (See https://timescaledb.slack.com/archives/C011FC0PPC5/p1653384295441989?thread_ts=1652775715.500199&cid=C011FC0PPC5)

File: CHANGELOG.md (hunk 1 above)
Comment (niksajakovljevic) [reply CHANGELOG.md]: I believe all paths are affected not only `/healthz`

**Comment by niksajakovljevic** [comment]
@sumerman Imho change log should contain notable changes and not bugs we discover later on. This is not something that we wanted for our users. Btw also note that this problem probably affects all endpoints

File: CHANGELOG.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ We use the following categories for changes:
 - The Promscale extension is now required, while the Timescaledb extension remains optional. The minimum Timescaledb version supported is now 2.6.1 [#1132], [#1297]
 - Disable running Promscale in HA and read-only simultaneously [#1254]
 - Metric tables and views are now owned by prom_admin [#1283]
+- The only HTTP verb `/healthz` endpoint supports is `GET`, because HTTP router was changed in [#1224].
```
[the remark this answers is not in the document]
Comment (Harkishen-Singh) [reply CHANGELOG.md:36]: But i agree what Niksa said

**Comment by niksajakovljevic** [comment]
A fix was done in https://github.com/timescale/promscale/pull/1378 . We will make a bugfix release

**Pull request closed without merging**